README.md

```diff
@@ -3,7 +3,12 @@ This repository contains code for interfacing with various robot platforms and o
 
 **Note** this repository is public so should only be used for utility code and example models - larger models should go in their own repositories and only be made public upon publication.
 
-# Prerequistites
+## Prerequistites
+A full list of requirements for different projects can be found [in the documentation](https://github.com/BrainsOnBoard/bob_robotics/documentation/requirements.md).
+
+The main requirements are as follows:
 * GeNN 3.X - Stable releases of GeNN can be downloaded from https://github.com/genn-team/genn/releases. However some models in this repository may require the 'development' branch from https://github.com/genn-team/genn/.
```

Comment:
I totally should have done this ages ago but this should refer to the 'master' branch (I managed to successfully escape Thomas's odd branching strategy since writing this)